apis/components/v1alpha1/datasciencepipelines/types.go

```diff
@@ -0,0 +1,21 @@
+// +kubebuilder:object:generate=true
+
+// Package datasciencepipelines provides a set of types used for DataSciencePipelines component
+package datasciencepipelines
+
+import operatorv1 "github.com/openshift/api/operator/v1"
+
+type PreloadedPipelinesSpec struct {
+	InstructLab InstructLabPipelineSpec `json:"instructLab,omitempty"`
+}
+
+type InstructLabPipelineSpec struct {
```

Comment:
Could you please make this a generic type for now? That way we can reuse it for future pipelines:
```suggestion
type PreloadedPipelineOptions struct {
```

Reply:
This is the specific IntructLab structure nested inside the PreloadedPipelinesSpec struct, isn't that enough ?

Reply:
I'm just saying that in the future, we can reuse the struct type if we add additional fields to `PreloadedPipelinesSpec`.

If we need options that are specific to the InstructLab preloaded pipeline, we can make it a specific struct but for now, I think all options will be shared across all preloaded pipelines.

Reply:
as far as I know, this was about enabling InstructLab, have things changed ?

Reply:
@lburgazzoli we expect to add additional preloaded pipelines in the future, but not right now. Just trying to make things more generic and easier for us in the future.

Reply:
done